com.sap.cloud.lm.sl.cf.persistence/src/test/java/com/sap/cloud/lm/sl/cf/persistence/util/DefaultConfigurationTest.java

```diff
@@ -0,0 +1,25 @@
+package com.sap.cloud.lm.sl.cf.persistence.util;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+
+import org.junit.jupiter.api.Test;
+
+import com.sap.cloud.lm.sl.cf.persistence.model.FileEntry;
+import com.sap.cloud.lm.sl.cf.persistence.processors.DefaultFileDownloadProcessor;
+import com.sap.cloud.lm.sl.cf.persistence.processors.DefaultFileUploadProcessor;
+import com.sap.cloud.lm.sl.cf.persistence.services.FileContentProcessor;
+
+public class DefaultConfigurationTest {
+
+    private DefaultConfiguration classUnderTest = null;
+
+    @Test
+    public void testNoAdditionalProcessingIsDone() {
+        classUnderTest = new DefaultConfiguration();
```

Comment:
Can this initialization be extracted in a BeforeEach method?

Reply:
done